packages/ckeditor5-dev-web-crawler/README.md

```diff
@@ -0,0 +1,18 @@
+CKEditor 5 travis status notifier
+================================
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-travis-status-notifier.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-travis-status-notifier)
+[![Build Status](https://travis-ci.com/ckeditor/ckeditor5-dev.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5-dev)
+![Dependency Status](https://img.shields.io/librariesio/release/npm/@ckeditor/ckeditor5-travis-status-notifier)
+
+Tasks used to build and verify the documentation for [CKEditor 5](https://ckeditor.com).
+
+More information about development tools packages can be found at the following URL: <https://github.com/ckeditor/ckeditor5-dev>.
+
+## Changelog
+
+See the [`CHANGELOG.md`](https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-travis-status-notifier/CHANGELOG.md) file.
+
+## License
+
+Licensed under the terms of [GNU General Public License Version 2 or later](http://www.gnu.org/licenses/gpl.html). For full details about the license, please check the `LICENSE.md` file.
```

Comment:
This `README.md` should be aligned to the `ckeditor5-dev-web-crawler` package, as it currently contains a lot of _travis status notifier_ mentions.